### ISSUE TITLE
Allow user to see their submitted expenses on the collective page ahead of their approvals

### ISSUE THREAD

**Issue by alinamanko** [opencollective/opencollective#5963]
https://opencollective.freshdesk.com/a/tickets/119774

## User story 
As a user, after I submit an expense it is not easy for me to see whether it has been successfully recorded on the collective page as there is currently no visibility of the expenses pending approval. The transactions listing on the user end only gets updated once the expense is approved. This lack of visibility can result in duplicated requests being submitted, increasing the risk of overpayments, if those are not caught by the collective and host admins. 

## Best solution for this problem
Can we add an UI where users can view submitted requests, so they know it has indeed been submitted pending approval?

 P3 high frequency, low impact

**Comment by Betree** [comment]
Duplicate of https://github.com/opencollective/opencollective/issues/3749
